app.js

```diff
@@ -1,1081 +1,15 @@
-const fs = require('fs');
-const path = require('path');
-const acorn = require('acorn');
-const escodegen = require('escodegen');
 const argv = require('boring')();
 const cp = require('child_process');
-const { stripIndent } = require('common-tags');
-
-// This is a random identifier not used anywhere else
-const blankLineMarker = '// X0k7FEu5a6!bC6mV';
+const linter = require('./lib/linter');
+const converter = require('./lib/converter');
 
 if (argv._[0] === 'reset') {
   cp.execSync('git reset --hard && git clean -df');
-  process.exit(0);
-}
-
-try {
-  cp.execSync('git status', { encoding: 'utf8' });
-} catch (e) {
-  fail(stripIndent`
-    This project does not appear to be using git. For your protection and to simplify
-    certain operations this tool can only be used in a git repository.
-  `);
-}
-
-if (isSingleSiteProject()) {
-  // Only touch directories, don't mess about with regular files in lib/modules (WTF)
-  // or symlinks in lib/modules (double WTF, but outside our remit)
-  let moduleNames = fs.readdirSync('lib/modules').filter(moduleName => fs.lstatSync(`lib/modules/${moduleName}`).isDirectory());
-  moduleNames.forEach(name => {
-    rename(`lib/modules/${name}`, `modules/${name}`);
-  });
-  try {
-    fs.rmdirSync('lib/modules');
-  } catch (e) {
-    if (e.code === 'ENOTEMPTY') {
-      console.error(stripIndent`
-        "lib/modules" is not empty after moving modules to "modules". You probably
-        have files that are not apostrophe modules in that folder. Please
-        move those files to a more appropriate location, like lib, then
-        remove "lib/modules" yourself.
-      `);
-    } else {
-      throw e;
-    }
-  }
-  moduleNames = fs.readdirSync('modules');
-  moduleNames.forEach(processModule);
-} else if (isSingleNpmModule()) {
-  const moduleName = process.cwd();
-  processModule(moduleName);
-} else if (isMoogBundle()) {
-  throw new Error('TODO: implement moog bundle migration');
+} else if (argv._[0] === 'lint') {
+  linter();
+} else if (argv._[0] === 'convert') {
```

Comment:
I would prefer not to use `run` because it more strongly implies you should definitely do it and for a lot of modules and some projects it just won't be a good fit. But I'm fine with `upgrade`. I'll go with `upgrade`.